package.xml

```diff
@@ -3,32 +3,27 @@
   <name>agimus_msgs</name>
   <version>0.0.1</version>
```

Comment:
```suggestion
  <version>0.0.2</version>
```
I think we can assume the version is 2-is now as this is a complete recreation of this package